site/docs/how-to-release.md

```diff
@@ -22,6 +22,17 @@ To create a release candidate, you will need:
 * Apache LDAP credentals for Nexus and SVN
 * A [GPG key for signing](https://www.apache.org/dev/release-signing#generate), published in [KEYS](https://dist.apache.org/repos/dist/dev/iceberg/KEYS)
 
+If you have not published your GPG key yet, you must publish it before sending the vote email by doing:
```

Comment:
Nit: Should we add a note that this is only required for PMC members (binding votes)? Some users might read through this and think they need to do it.

Up to you.

Reply:
this is needed for anyone managing the release, not just PMC member. People validating the release artifact signature are supposed to import keys through the KEYS file.

Reply:
Agreed. This is needed for any RM to sign the artifacts.